test/testvm.py

```diff
@@ -776,6 +834,8 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
             "-kernel", self._image_kernel,
             "-initrd", self._image_initrd
         ]
+        if self.init_image:
+            cmd += ["-cdrom", self.init_image]
```

Comment:
Does every boot of Atomic need the init ISO?  Or do we only need it during vm-create?

I was hoping that fedora-atomic-22.bootstrap would end with a useable <flavor>-fedora-atomic-22-x86_64.qcow2 which is ready to run the setup script.